jwt/jwt.go

```diff
@@ -134,58 +134,58 @@ func TokenSerialize(t jwt.Token) ([]byte, error) {
 //
 // Header:
 //
-// {
-//  "alg": "RS256",
-//  "kid": "a2aa9739-d753-4a0d-87ee-61f101050277",
-//  "typ": "JWT"
-// }
+//	{
```

Comment:
This whole formatting thing happens during our linting run. I'll dig into it later.